src/soundio/soundmanagerconfig.cpp

```diff
@@ -513,9 +514,10 @@ void SoundManagerConfig::loadDefaults(SoundManager* soundManager, unsigned int f
     if (flags & SoundManagerConfig::DEVICES) {
         clearOutputs();
         clearInputs();
-        QList<SoundDevicePointer> outputDevices = soundManager->getDeviceList(m_api, true, false);
+        const QList<SoundDevicePointer> outputDevices =
+                soundManager->getDeviceList(m_api, true, false);
```

Comment:
Have you made sure to double check that these are only applied to instances where you return a `const&`? If they return proper values, making them const may lead to other inefficiencies later (disabling move semantics primarily).

Reply:
It should be only for the QLists, where Clazy recommended std::as_const and the list is never modified.